FormattableTextViewTests/FormattableTextTests.swift

```diff
@@ -58,4 +58,11 @@ class FormattableTextTests: XCTestCase {
 		XCTAssertEqual(result, "t(459-_*asdf")
 	}
 
+	func testNonFormattedTextInTextView() throws {
```

Comment:
Thanks for the PR!
Could you add the same test for FormattableTextField?

Reply:
@zlib done